fastlane/lib/fastlane/actions/download_dsyms.rb

```diff
@@ -141,12 +163,17 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :version,
                                        short_option: "-v",
                                        env_name: "DOWNLOAD_DSYMS_VERSION",
-                                       description: "The app version for dSYMs you wish to download",
+                                       description: "The app version for dSYMs you wish to download, pass in 'latest' to download only the latest build's dSYMs",
                                        optional: true),
           FastlaneCore::ConfigItem.new(key: :build_number,
                                        short_option: "-b",
                                        env_name: "DOWNLOAD_DSYMS_BUILD_NUMBER",
                                        description: "The app build_number for dSYMs you wish to download",
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :save_directory,
+                                       short_option: "-s",
+                                       env_name: "DOWNLOAD_DSYMS_SAVE_DIRECTORY",
```

Comment:
Could we change this to be called `output_directory` to match some of our other tools with similar options?

Reply:
updated 👍